src/hexo/themes/sonarwhal/helper/index.js

```diff
@@ -269,6 +269,11 @@ module.exports = function () {
         isNotSectionIndexPage: (page) => {
             return !isSectionIndexPage(page);
         },
+        isPass: (status, overralstatistics) => {
+            const passStatistics = (overralstatistics.errors === 0) && (overralstatistics.warnings === 0);
```

Comment:
@qzhou1607 Is there a reason why you use `Statistics` in the names?

Reply:
@alrra No particular reasons, just inheriting the variable name from the server side. What variable name do you prefer that I use instead?

Reply:
>  What variable name do you prefer that I use instead?

Maybe `stats` (`overralStats`)?